artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -909,10 +943,25 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
-          example:
-            status: 409
-            code: CONFLICT
-            message: "The specified resource is in a conflict"
+          examples:
+            GENERIC_409_ABORTED:
+              description: Concurreny of processes of the same nature/scope
+              value:
+                status: 409
+                code: ABORTED
+                message: Concurrency conflict.
+            GENERIC_409_ALREADY_EXISTS:
```

Comment:
I would keep this and remove GENERIC_409_CONFLICT, at the end covers the same scenario in the context of this API template

Reply:
Yes this is right. I fix this.